src/internal/secrets-manager.ts

```diff
@@ -1,7 +1,7 @@
 import { JSONArray, JSONObject } from 'k6'
 import http, { RefinedResponse, ResponseType } from 'k6/http'
+import { crypto } from 'k6/experimental/webcrypto'
```

Comment:
Just as a case report. We don't expect to remove the WebCrypto module, but here we are building a product on top of an experimental feature.

Reply:
Yeah, I agree that technically we shouldn't do so unless we consider `k6-jslib-aws` also experimental. With this, if k6 users start using the next release of this library and, at some point, we move `webcrypto` to _stable_, they'll start getting warnings, that aren't directly produced by their code.

To be fair, I don't think there's such a big deal here, but definitely something that worths agreement within the team, and to set a precedent (guidelines) for future cases.

Reply:
> they'll start getting warnings

This is a good catch. It sounds relevant enough to roll back the change. Considering that we already have a working alternative to it.

Reply:
Good point folks 👍🏻 I switched to k6-utils' `uuidv4` in the meantime.